absl/time/internal/cctz/src/time_zone_if.cc

```diff
@@ -37,7 +37,7 @@ std::unique_ptr<TimeZoneIf> TimeZoneIf::Load(const std::string& name) {
 }
 
 // Defined out-of-line to avoid emitting a weak vtable in all TUs.
-TimeZoneIf::~TimeZoneIf() {}
+TimeZoneIf::~TimeZoneIf() = default;
```

Comment:
The changes to all files under `absl/time/internal/cctz/` need to be sent to https://github.com/google/cctz. We will then import these changes.